src/cli.ts

```diff
@@ -80,19 +80,22 @@ program
       Object.keys(reporters)
     )
   )
-  .option('--inline', 'Run inline journeys from heartbeat')
+  .option('--inline', 'read journeys from stdin instead of reading from files')
   .option('-r, --require <modules...>', 'module(s) to preload')
   .option('--sandbox', 'enable chromium sand-boxing')
-  .option('--rich-events', 'Mimics a heartbeat run')
-  .option('--no-headless', 'run browser in head-full mode')
+  .option(
+    '--rich-events',
+    'preset flag used when running monitors directly via Heartbeat'
+  )
+  .option('--no-headless', 'run with the browser in head-full mode')
```

Comment:
```suggestion
  .option('--no-headless', 'run with the browser in headful mode')
```

This is really jargon-y but I think `headful` is an accepted term for this meaning. I realize it deviates from what we had before but I think this is allowed and nicer.